Don't find service in the specified "import" path

I try to import the "UrlParsingService" in my customized service but the path shows "@spartacus/core/src/routing/configurable-routes/url-translation/url-parsing.service"; and the console shows "Module not found: Error: Can't resolve '@spartacus/core/src/routing/configurable-routes/url-translation/url-parsing.service' in 'xxx/protected-routes'

<img width="1642" alt="Screen Shot 2021-08-18 at 2 28 21 PM" src="https://user-images.githubusercontent.com/40365471/129960602-27b687da-e7f6-4473-b2b9-14eea27188d7.png">


**Desktop
 - OS: [MacOs]
 - Browser [chrome]
 - Version [22]

--- a/projects/core/src/routing/configurable-routes/url-translation/index.ts
+++ b/projects/core/src/routing/configurable-routes/url-translation/index.ts
@@ -1,5 +1,6 @@
 export * from './product-url.pipe';
 export * from './semantic-path.service';
 export * from './url-command';
+export * from './url-parsing.service';
 export * from './url.module';
 export * from './url.pipe';

